sympy/physics/optics/tests/test_polarization.py

```diff
@@ -1,7 +1,7 @@
 from sympy.physics.optics.polarization import (jones_vector, stokes_vector,
     jones_2_stokes, linear_polarizer, phase_retarder, half_wave_retarder,
     quarter_wave_retarder, transmissive_filter, reflective_filter,
-    mueller_matrix, polarizing_beam_splitter)
+    mueller_matrix, polarizing_beam_splitter, quarter_wave_plate)
 from sympy.core.numbers import (I, pi)
 from sympy.core.singleton import S
 from sympy.core.symbol import symbols
```

Comment:
This line seems redundant

Reply:
Sir, can you explain how this line seems redundant?